tests/test.js

```diff
@@ -1,50 +1,120 @@
 require('dotenv').config({ path: './.env'});
```

Comment:
distribute to tests to various files
like`tests/util-test.js`, `tests/backend-test.js` etc. I am not sure about the naming convention though. Please refer any relevant resource for that